src/pybind/mgr/requirements-required.txt

```diff
@@ -13,3 +13,4 @@ pyyaml
 requests-mock
 scipy
 werkzeug
+natsort
```

Comment:
looks like we need similar for the dashboard tox tests. They are failing with

```
Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 185, in _run_module_as_main
    mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
  File "/usr/lib/python3.8/runpy.py", line 111, in _get_module_details
    __import__(pkg_name)
  File "/home/jenkins-build/build/workspace/ceph-pull-requests/src/pybind/mgr/dashboard/__init__.py", line 52, in <module>
    from .module import Module, StandbyModule  # noqa: F401
  File "/home/jenkins-build/build/workspace/ceph-pull-requests/src/pybind/mgr/dashboard/module.py", line 31, in <module>
    from .services.exception import dashboard_exception_handler
  File "/home/jenkins-build/build/workspace/ceph-pull-requests/src/pybind/mgr/dashboard/services/exception.py", line 10, in <module>
    from orchestrator import OrchestratorError
  File "/home/jenkins-build/build/workspace/ceph-pull-requests/src/pybind/mgr/orchestrator/__init__.py", line 3, in <module>
    from .module import OrchestratorCli
  File "/home/jenkins-build/build/workspace/ceph-pull-requests/src/pybind/mgr/orchestrator/module.py", line 10, in <module>
    from natsort import natsorted
ModuleNotFoundError: No module named 'natsort'
```

```
/usr/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
__init__.py:52: in <module>
    from .module import Module, StandbyModule  # noqa: F401
module.py:31: in <module>
    from .services.exception import dashboard_exception_handler
services/exception.py:10: in <module>
    from orchestrator import OrchestratorError
../orchestrator/__init__.py:3: in <module>
    from .module import OrchestratorCli
../orchestrator/module.py:10: in <module>
    from natsort import natsorted
E   ModuleNotFoundError: No module named 'natsort'
```

(see make check logs)